docker/files/etc/nginx/conf.d/etherpad.conf

```diff
@@ -0,0 +1,57 @@
+upstream etherpad-backend {
+  server etherpad:9001 fail_timeout=0;
+}
+
+server {
+  listen 9001;
+  server_name localhost;
+
+  client_max_body_size 100M;
+
+  rewrite ^(.*)/favicon.ico$ /static/images/favicon.ico last;
+
+  # Disables server version feedback on pages and in headers
+  server_tokens off;
+
+  location @proxy_to_etherpad_app {
+    proxy_set_header Host $http_host;
+    proxy_pass_header Server;
+    # be careful, this line doesn't override any proxy_buffering on set in a conf.d/file.conf
+    proxy_buffering off;
+    # http://wiki.nginx.org/HttpProxyModule
+    proxy_set_header X-Real-IP $remote_addr;
+    # EP logs to show the actual remote IP
+    proxy_set_header X-Forwarded-For $remote_addr;
+    # for EP to set secure cookie flag when https is used
+    proxy_set_header X-Forwarded-Proto $scheme;
+    # pass the host header
+    proxy_set_header Host $host;
+    # recommended with keepalive connections
+    proxy_http_version 1.1;
+    # WebSocket proxying - from http://nginx.org/en/docs/http/websocket.html
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection $connection_upgrade;
+    proxy_redirect off;
+
+    # Required to resolve the scaled backend application
+    resolver 127.0.0.11 valid=5s;
```

Comment:
Is this valid in a docker context ? Should not be `resolver etherpad` ? Using an ip address is a little bit strange don't you think ?

Reply:
This is required for `nginx` to resolve `etherpad` in a docker-compose context. `127.0.0.11` is the IP of the docker network internal DNS.

Reference: https://nginx.org/en/docs/http/ngx_http_core_module.html#resolver

Reply:
:ok: